client/components/Toggle/index.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import classNames from 'classnames'
+import './style.scss'
+
+export default function Toggle({ value, onChange, readOnly }) {
+    const onClick = () => onChange({ target: { value: !!value } })
```

Comment:
Yes I think I even did this typo myself. And I never noticed it because I wasn't using the onChange method I guess. But now you are using it, I think it makes sense to return the new value as parameter ?